wellregistry/postgres/migrations/0001_create_db_users.py

```diff
@@ -12,7 +12,7 @@
 
 from django.db import migrations
 from django.conf import settings
-import postgres.pgsql_utils as pgsql
+import wellregistry.pgsql_utils as pgsql
```

Comment:
I moved this out of postgres app as it seemed like more generally reusable  methods. By moving it out the registry app is not dependent on the postgres app either which may be a good thing.